middleware/slow_weblogic_start.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Slow WebLogic Starts
+layout: en
+permalink: /posts/middleware/slow_weblogic_starts/
+---
+
+If Weblogic is booting slowly on Linux, it might be a low entropy situation. If entropy is less than 1000, 
+
+```bash
+cat /proc/sys/kernel/random/entropy_avail
```

Comment:
This might be better as a `watch -n 1 /proc/sys/kernel/random/entropy_avail` during startup, rather than a single sample.

Reply:
Good idea.